### PR TITLE
chore: fix dep alerts

### PR DESCRIPTION
**Pull request by souradeep-das**

### CODE REVIEW

File: plasma_framework/package.json [hunk 1]
```diff
@@ -27,14 +27,15 @@
...
     "solhint": "^2.2.0",
-    "solidity-coverage": "^0.7.2",
-    "truffle": "5.1.36"
+    "solidity-coverage": "^0.7.12",
+    "truffle": "^5.1.48"
   },
   "scripts": {
     "test": "truffle test",
-    "coverage": "truffle run coverage",
+    "coverage": "node_modules/.bin/truffle --max-old-space-size=4096 run coverage",
```
Comment (kevsul) [inline plasma_framework/package.json:38]: Were there out of memory errors when running coverage?

**Review by kevsul — APPROVED** [review]

File: plasma_framework/package.json (hunk 1 above)
Comment (souradeep-das) [reply plasma_framework/package.json:38]: yes, `heap out of memory` after I bumped coverage to latest.

**Comment by souradeep-das** [comment]
`node-fetch` that comes with hdwallet is still not resolved

**Comment by InoMurko** [comment]
This is closing https://github.com/omgnetwork/plasma-contracts/issues/703 ? correct? I linked it.

**Review by InoMurko — APPROVED** [review]

**Comment by souradeep-das** [comment]
yep, thanks 👍. Doing one final check before merging this

**Pull request merged**